LandSandBoat/server
# [WS] Add multihit FTP to Axe weaponskills

**_I affirm:_**

## What does this pull request do?

- Adds params.multiHitfTP to Axe weaponskills Decimation, Rampage, and Ruinator.
- Adjusts formatting of the associated Axe weaponskills to conform to the 120 character per line limit.

Reference:
https://www.bg-wiki.com/ffxi/Category:FTP_Replicating_WS

- Issue #1357 lists the weaponskills found to be missing the multiHitfTP code.
- PR #1355 addresses H2H weaponskills missing multiHitfTP
- PR #1380 addressed Dagger weaponskills missing multiHitfTP.



## Steps to test these changes

Load the adjusted weaponskill LUA into your server.
Pick your favorite mob to test against and use the weaponskills.
fTP adjusting items like the Fotia Gorget and Belt should make significant differences in the output of these weaponskills.

Useful commands:
!tp 1000-3000 charName
